frontends/election-manager/src/screens/write_ins_transcription_screen.tsx

```diff
@@ -86,8 +89,10 @@ export function WriteInsTranscriptionScreen({
 }: {
   contest: CandidateContest;
   election: Election;
-  onClickNext: () => void;
-  onClickPrevious: () => void;
+  ballotIdxBeingAdjudicated: number;
+  ballotsBeingAdjudicated: CastVoteRecord[];
```

Comment:
We don't need to mutate this, right?
```suggestion
  ballotsBeingAdjudicated: readonly CastVoteRecord[];
```